browser/src/control/jsdialog/Widget.TreeView.ts

```diff
@@ -563,17 +566,17 @@ class TreeViewControl {
 				cell.innerText = text;
 			}
 
-			if (hasRenderer) {
-				JSDialog.OnDemandRenderer(
-					builder,
-					treeViewData.id,
-					'treeview',
-					entry.row,
-					cell,
-					parent,
-					entry.text,
-				);
-			}
+		// in case of non-persistent entries we want to re-render in case of change
+		if (hasRenderer) {
```

Comment:
note: OnDemandRenderer has "persistent" detection inside to not request render again